app/assets/stylesheets/content/_notifications.sass

```diff
@@ -49,6 +49,8 @@ $nm-color-warning-border: #b5b078
 $nm-color-warning-icon: #000
 $nm-color-warning-background: #f4f4aa
 
+$nm-color-info-icon: #000
```

Comment:
I see that the warning icon also has `#000`, but I don't quite understand why those icons have a different color than the text. (because the difference is really marginal)